exdb/views.py

```diff
@@ -45,3 +46,24 @@ class PendingApprovalQueueView(ListView):
 
     def get_queryset(self):
         return Experience.objects.filter(status='pe')
+
+
+class RAHomeView(ListView):
+    template_name = 'exdb/ra_home.html'
+    context_object_name = 'experiences'
+
+    def get_queryset(self):
+        return Experience.objects.filter(author=self.request.user).order_by('created_datetime')
+
+    def get_context_data(self, *args, **kwargs):
+        context = super(RAHomeView, self).get_context_data(*args, **kwargs)
+        context['ra'] = self.request.user
+
+        experience_dict = OrderedDict()
+        for status in Experience.STATUS_TYPES:
+            experience_dict[status[1]] = []
+        for experience in context[self.context_object_name]:
+            experience_dict[experience.get_status_display()].append(experience)
```

Comment:
Perhaps this should be something like 
```python
experience_dict = {}
for experience in context[self.context_object_name]:
    experience_dict.setdefault(experience.get_status_display(), []).append(experience)
context['experience_dict'] = experience_dict
```
This will require changes in the template.
Then again, perhaps not.